### ISSUE TITLE
[Regression] aab51f2 fails make test

### ISSUE THREAD

**Issue by protist** [bit-team/backintime#725]
I'm trying to install backintime from Arch Linux's AUR, however the latest version `11.1.12.r370.gc5cfedf51` fails the tests.

I downloaded sources and built manually. The previous version 0d7d4608033df29ecdeeeae6aff45c47264056da passes `make test` fine, but the next commit aab51f2226e0735d7e6dbb6d80505a06e3cdef58 fails as below.

```
$ cd common
$ ./configure --python3 --no-fuse-group
...
$ make
...
$ LC_ALL=en_US.UTF-8 make test

python3 -m unittest  -b test/test_applicationinstance.py
...................
----------------------------------------------------------------------
Ran 19 tests in 0.051s

OK
python3 -m unittest  -b test/test_argparser.py
.....................
----------------------------------------------------------------------
Ran 21 tests in 5.198s

OK
python3 -m unittest  -b test/test_backintime.py
..
----------------------------------------------------------------------
Ran 2 tests in 4.446s

OK
python3 -m unittest  -b test/test_backup.py
............
----------------------------------------------------------------------
Ran 12 tests in 1.124s

OK
/usr/lib/python3.6/subprocess.py:761: ResourceWarning: subprocess 14650 is still running
python3 -m unittest  -b test/test_configfile.py
WARNING: Failed to read process stat from /proc/14605/stat: [2] No such file or directory
WARNING: Failed to read process stat from /proc/14791/stat: [2] No such file or directory
WARNING: Failed to read process stat from /proc/14792/stat: [2] No such file or directory
WARNING: Failed to read process stat from /proc/14793/stat: [2] No such file or directory
......................................................
----------------------------------------------------------------------
Ran 54 tests in 0.012s

OK
python3 -m unittest  -b test/test_config.py
..........
----------------------------------------------------------------------
Ran 10 tests in 0.025s

OK
python3 -m unittest  -b test/test_encfstools.py
.
----------------------------------------------------------------------
Ran 1 test in 0.000s

OK
python3 -m unittest  -b test/test_gocryptfstools.py
E
======================================================================
ERROR: test_init (test.test_gocryptfstools.TestGoCryptFS_mount)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/backintime/common/test/test_gocryptfstools.py", line 35, in test_init
    self.mnt.init()
  File "/tmp/backintime/common/gocryptfstools.py", line 97, in init
    universal_newlines = True)
  File "/usr/lib/python3.6/subprocess.py", line 707, in __init__
    restore_signals, start_new_session)
  File "/usr/lib/python3.6/subprocess.py", line 1326, in _execute_child
    raise child_exception_type(errno_num, err_msg)
FileNotFoundError: [Errno 2] No such file or directory: 'gocryptfs'

----------------------------------------------------------------------
Ran 1 test in 0.013s

FAILED (errors=1)
```

**Comment by Germar** [comment]
I accidentally committed a dev branch into master. Will remove it later today. Sorry

**Comment by willemw12** [comment]
Before I read Germar's comment:

In the Arch Linux AUR:

Fixed the build test by adding 'gocryptfs' as a dependency. However, in a clean chroot build environment, the gocryptfs test hangs:

    python3 -m unittest  -b test/test_gocryptfstools.py
    E

**Comment by Germar** [comment]
`gocryptfs` support is not ready yet. Just some rudimentary functions

**Comment by Germar** [comment]
fixed in 87e7a6fb548aa91c3ac0be2fd59cd0e4bbd8a2b4

**Comment by protist** [comment]
No worries @Germar! Thanks for the quick fix!
